bigbluebutton-html5/imports/ui/components/breakout-room/breakout-room/styles.ts

```diff
@@ -345,6 +341,35 @@ const ErrorMessage = styled.div`
   max-height: .93rem;
 `;
 
+const NotAssignedWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  background-color: ${colorBlueLightest};
+  border-radius: ${borderRadius};
```

Comment:
The suggested border is 0.5rem, currently it is 0.2rem

```suggestion
  border-radius: 0.5rem;
```